.github/workflows/build.yml

```diff
@@ -0,0 +1,62 @@
+name: SailfishOS build
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  clang-format-checking:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: RafikFarhad/clang-format-github-action@v1
+        with:
+          sources: "src/**/*.h,src/**/*.c,test/**/*.c"
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Prepare
+      run: mkdir output
+
+    - name: Build armv7hl
+      id: build_armv7hl
+      uses: coderus/github-sfos-build@master
+      with:
+        release: 4.0.1.48
```

Comment:
Can I ask where this specific version number came from? I only see 4.0.1.45 in the [tags](https://github.com/CODeRUS/docker-sailfishos-platform-sdk/tags) for CODeRUS's docker-sailfishos-platform-sdk docker image source repo.

Reply:
I think I used the release SFOS 4 version number, because I saw that it will just pull whatever version from https://releases.jolla.com. But I didn't checked if all the repos for 4.0.1.48 actually exist.

Reply:
So, the number here doesn't really matter?

Reply:
Afaik it has to be a valid Sailfish releases where Jolla provides a repo for. The tags only show which version is capable of building SFOS $version with changed build conventions.  

And GH Actions seem to currently use the master branch  of the repo.

Reply:
Okay, great, thanks for clarifying that for me.